draft-ietf-core-sid.md

```diff
@@ -127,6 +134,10 @@ modules and associated SIDs. To enable the implementation of this registry,
 {{sid-file-format}} defines a standard file format used to store and publish
 SIDs.
 
+For IETF managed YANG modules, the allocation of SIDs via the IANA mechanism specified in this document is MANDATORY.
```

Comment:
I am a bit worried that _MANDATORY_ here might face some opposition (what about old ones), although it will probably make everybody's life easier. I believe the general idea so far was allocating SIDs at module creation or when needed.

Reply:
I think that the sentence is pretty clear.  If you want to allocate SIDs, AND it's an IETF yang module, then you have to go to IANA.  (I thought that MANDATORY was a synonym for MUST, but BCP14 disagrees.  I will rewrite to say MUST).
If old modules don't need SID values, then they won't get them.

Reply:
In my mind, we are straddling the boundary between the two documents (yang-cbor, sid) here again.